src/main/js/components/header/Spinner.jsx

```diff
@@ -79,16 +79,18 @@ export default class Spinner extends React.Component {
         </div>
       );
     } else {
-      return null;
+      return <div/>;
```

Comment:
Why?

Reply:
Else we will have an empty children prop in the portal-hoc, crashing the whole app